addons/friendly_tracker/functions/fnc_loop.sqf

```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3, veteran29
+ * BFT loop
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call afm_friendly_tracker_fnc_loop
+ *
+ * Public: No
+ */
+
+// Check if BFT is disabled
+if (!GVAR(enabled)) exitWith {
+    [QGVAR(stop)] call CBA_fnc_localEvent;
+};
+
+// Remove all markers
+{
+    deleteMarkerLocal (_x select 0);
+} foreach GVAR(markers);
+
+// If GPS mode is enabled and player (local) does not have GPS/UAV terminal we break and schedule next loop
+if (GVAR(GPS) && {!([player] call FUNC(hasTracker))}) exitWith {
+    [FUNC(loop), [], GVAR(refreshRate)] call CBA_fnc_waitAndExecute;
+};
+
+// Create marker for every player in game
+{
+    call {
```

Comment:
What's the reason for additional call inside the loop?

Reply:
It adds nesting for exitWith when player does not have a tracker.

Reply:
Oh, haven't noticed. `exitWith {}` will stop the loop if not wrapped, however changing it to `if then` inside the loop will be faster.

Reply:
Already did that. I thought I will do something more here, but as it ended like this it will be better with `if then`.